utils.py

```diff
@@ -108,7 +108,7 @@ def weather(cmd):
 	else:
 		flags = '1Fp'
 		location = cmd.args
-	filename = '%s_%s.png' % (urllib.parse.quote_plus(location), flags)
+	filename = '%s.png?%s' % (urllib.parse.quote_plus(location), flags)
 	url = 'https://wttr.in/' + filename
```

Comment:
it's not really a filename any more, so how about
```suggestion
	url = 'https://wttr.in/%s.png?%s' % (urllib.parse.quote_plus(location), flags)
```